shell/src/main/java/alluxio/cli/fs/command/UpdateMountCommand.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fs.command;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileSystemContext;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.grpc.MountPOptions;
+
+import com.google.common.collect.Maps;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * Update options for an Alluxio mount point.
+ */
+@ThreadSafe
+public final class UpdateMountCommand extends AbstractFileSystemCommand {
```

Comment:
I thought about it and there are a couple reasons for using a separate command:
- In mount command the user has to specify a UFS URI; That is not necessary for updateMount.
- We already have `mount` and `unmount ` instead of a single command with `-u` option. In the spirit of "Do One Thing And Do It Well", I made it a separate command.
- It is harder to describe the command and usage if we have the mount command and updateMount command combined.
- It's easier to maintain if we later wants to add options to only one of the command.
What do you think?

Reply:
Ok, sounds good.